go-controller/pkg/ovn/pods.go

```diff
@@ -243,7 +244,7 @@ func (oc *Controller) addLogicalPort(pod *kapi.Pod) error {
 		// If the pod already has annotations use the existing static
 		// IP/MAC from the annotation.
 		args = append(args,
-			"--", "lsp-set-addresses", portName, fmt.Sprintf("%s %s", podMac, annotation.IP.IP),
+			"--", "lsp-set-addresses", portName, fmt.Sprintf("%s %s", podMac, podCIDR.IP),
```

Comment:
@danwinship I think you can just do something like:

```
var addrs []string
for _, podCIDR := range annotation.IPs {
    addrs = append(addrs, podCIDR.IP.String())
}
...
"--", "lsp-set-addresses", portName, fmt.Sprintf("%s %s", podMac, strings.Join(addrs, " ")),
```

Reply:
You mean to fix the FIXME? There's more to it than that though. `podCIDR` gets used again later in the function. It will all get fixed up in a future PR. I have to introduce arbitrary stopping points like this or else all of dual-stack ends up being a single PR.

Reply:
Yeah, good point, figured that out later in the review.